pychunkedgraph/ingest/ingestion_utils.py

```diff
@@ -2,22 +2,31 @@
 from pychunkedgraph.backend import chunkedgraph, chunkedgraph_utils
 
 import cloudvolume
+import collections
 
 
-def calc_n_layers(ws_cv, chunk_size, fan_out):
+def get_layer_count(ws_cv, chunk_size, fan_out):
     bbox = np.array(ws_cv.bounds.to_list()).reshape(2, 3)
     n_chunks = ((bbox[1] - bbox[0]) / chunk_size).astype(np.int)
-    n_layers = int( np.ceil(chunkedgraph_utils.log_n(np.max(n_chunks), fan_out))) + 2
+    n_layers = int(np.ceil(chunkedgraph_utils.log_n(np.max(n_chunks), fan_out))) + 2
     return n_layers
 
 
-def initialize_chunkedgraph(cg_table_id, ws_cv_path, chunk_size, size,
-                            cg_mesh_dir, use_skip_connections=True,
-                            s_bits_atomic_layer=None,
-                            n_bits_root_counter=8, fan_out=2, instance_id=None,
-                            project_id=None):
+def initialize_chunkedgraph(
+    cg_table_id,
+    ws_cv_path,
+    chunk_size,
+    cg_mesh_dir="mesh_dir",
+    size=None,
```

Comment:
Keep order constant

Reply:
I changed the whole thing, since its only initial ingestion I think this should be fine